src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { classNames } from '../../utils/classNames';
+
+// Animation types for the skeleton component.
+export type SkeletonAnimation = 'pulse'
+    | 'wave'
+    | false;
+
```

Comment:
consider adding Error Handling (? - just my perspective) toward handling valid animations and variants // check if props are valid

const validatedAnimation .... validatevariant -